### PR TITLE
Fix non admin logouts

### PR DESCRIPTION
**Pull request by harikrishna-patnala**
### Description

If a user (non-admin) logs out from a session, then login page is not loading completely. Few starter APIs like listIds are failing and showing unauthorised access notification in Login Page. Also if SAML is enabled, it is not getting enabled since the corresponding API are failed. User needs to refresh the browser to get it back.

image

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
Login and logout with non admin users and you dont see any errors in the login page and all applicable login options will be enabled
image

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
Didn't test it but LGTM

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

File: ui/src/views/dashboard/UsageDashboard.vue [hunk 1]
```diff
@@ -143,7 +143,9 @@ export default {
         if (newValue && newValue.id && (!oldValue || newValue.id !== oldValue.id)) {
           this.fetchData()
         } else if (store.getters.userInfo.roletype !== 'Admin') {
```
Comment (BryanMLima) [inline ui/src/views/dashboard/UsageDashboard.vue:145]:
```suggestion
        } else if (store.getters.userInfo.roletype !== 'Admin' && !store.getters.logoutFlag) {
```
Just a minor suggestion to decrease indentation.

**Review by BryanMLima — APPROVED** [review]
LGTM, manually tested this with a user type account. 

Screenshot from 2023-10-10 16-51-38

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan LLtest

**Comment by rohityadavcloud** [comment]
Since this is strictly UI only change smoketest wasn't required @DaanHoogland

**Pull request merged**